tests/test_fellow.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import nose
+
+from models.fellow import Fellow
+
+class TestFellow(unittest.TestCase):
+	"""Test cases for the Fellow class"""
+
+	def setUp(self):
+		"""Setup a new Fellow object to be used for testing"""
+		self.fellow = Fellow('Brian','Mosigisi')
+
+	def test_fellow_instance(self):
+		"""Tests if the object created is an instance of the Fellow class"""
+		self.assertIsInstance(self.fellow,Fellow)
+
+	def test_fellow_type(self):
+		"""Tests if the object created is of type Fellow"""
+		self.assertEqual(type(self.fellow),Fellow)
+
+	def test_person_type(self):
+		"""Tests if the Person type is Staff """
+		self.assertEqual(self.fellow.person_type,'FELLOW')
+
+	def test_person_name(self):
+		"""Tests if the full name is a combination of the first name and last name"""
+		self.assertEqual(self.fellow.name,'BRIAN MOSIGISI')
+
+	def test_person_office_allocated(self):
+		"""Tests if at creation, a new staff has no office allocated"""
+		self.assertEqual(self.fellow.office_allocated,False)
+
+	def test_person_office(self):
+		"""Tests if the Fellow created has no office"""
+		self.assertEqual(self.fellow.office,'')
+
+	def test_person_livingspace_allocated(self):
+		"""Tests if at creation, a new Fellow has no Living Space allocated"""
+		self.assertEqual(self.fellow.livingspace_allocated,False)
+
+	def test_person_livingspace(self):
+		"""Tests if the Fellow created has no Living Space"""
+		self.assertEqual(self.fellow.livingspace,'')
+
+	def test_check_fellow_id(self):
+		"""Check if the id assigned to a fellow is unique to only fellows"""
+		first_char = self.fellow.fellow_id[0]
+		self.assertEqual(first_char,'F')
```

Comment:
Am curious as to why you are only checking if the first character of the `id` is an `F`. Why not create a couple of Fellow objects and check whether their ids are not identical? What do you think about that?